src/Crypt32.Shared/Crypt32+CRYPT_KEY_PROV_INFO.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    public partial class Crypt32
+    {
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        public struct CRYPT_KEY_PROV_INFO
```

Comment:
[Our policy is to use pinnable structs](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#struct-field-types). Can you correct?